test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1228,6 +1228,61 @@ UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_wireup_unified, rc, "rc")
 UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_wireup_unified, ud, "ud")
 UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_wireup_unified, rc_dc, "rc,dc")
 
+class select_ep_transport : public test_ucp_wireup {
+public:
+    static void get_test_variants(std::vector<ucp_test_variant>& variants)
+    {
+        add_variant_with_value(variants, UCP_FEATURE_RMA, TEST_RMA, "rma");
+    }
+
+    char *get_selected_transport(const entity *e)
+    {
+        const ucp_worker *worker    = e->worker();
+        ucp_lane_index_t lane_index = worker->ep_config[0].key.rma_lanes[0];
+        ucp_rsc_index_t rsc_index   =
+                worker->ep_config[0].key.lanes[lane_index].rsc_index;
```

Comment:
we should use endpoint's config, rather than worker->ep_config[0]
`ucp_ep_config(e->ep())`